### ISSUE TITLE
No pin for Lambda and success/fail branches

### ISSUE THREAD

**Issue by barn4k** [n8n-io/n8n#10240]
### Bug Description

If the AWS Lambda has been set to **On Error - Continue Using Error output**, then the pin button will disappear. However it is still possible to pin data with Ctrl+P, but there is no option to edit it anymore
![image](https://github.com/user-attachments/assets/1ee0ae25-17e2-45fb-beaf-5638e8b0726b)

![image](https://github.com/user-attachments/assets/218fbd09-ab27-45fb-899f-2a842d36bf22)

### To Reproduce

Set the AWS Lambda On Error setting to "Continue (using error output)"
Try to find the pin data button :)

```
{
  "meta": {
    "instanceId": "ea8ccd7cb9d9fc233a4683c7569b952c9669ba9d3ba1459d89a9944d357fa5b3"
  },
  "nodes": [
    {
      "parameters": {},
      "id": "378c832a-30fb-48d7-9d90-94e9a89a2b27",
      "name": "When clicking ‘Test workflow’",
      "type": "n8n-nodes-base.manualTrigger",
      "typeVersion": 1,
      "position": [
        200,
        320
      ]
    },
    {
      "parameters": {},
      "id": "3e047b28-efb6-4503-ac52-9abfd1a3e3c4",
      "name": "AWS Lambda",
      "type": "n8n-nodes-base.awsLambda",
      "typeVersion": 1,
      "position": [
        420,
        320
      ],
      "onError": "continueErrorOutput"
    }
  ],
  "connections": {
    "When clicking ‘Test workflow’": {
      "main": [
        [
          {
            "node": "AWS Lambda",
            "type": "main",
            "index": 0
          }
        ]
      ]
    }
  },
  "pinData": {
    "AWS Lambda": [
      {
        "name": "First item",
        "code": 1
      },
      {
        "name": "Second item",
        "code": 2
      }
    ]
  }
}
``` 

### Expected behavior

The pin button works as expected

### Operating System

Cloud / Docker

### n8n Version

1.50.1, 1.49.1

### Node.js Version

dunno

### Database

PostgreSQL

### Execution mode

main (default)

**Comment by Joffcom** [comment]
Hey @barn4k,

This looks to be the same as https://github.com/n8n-io/n8n/issues/10066, This is down to where we don't support pinned data for multi output nodes and when we added the new error output we didn't update this.

As this looks to be a duplicate I am going to close this, Let me know if you have any questions on this.

**Comment by janober** [comment]
Fix got released with [`n8n@1.68.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.68.0)
